plugins/aea-ledger-fetchai/aea_ledger_fetchai/_cosmos.py

```diff
@@ -1225,8 +1210,8 @@ def get_transaction(self, tx_digest: str) -> Optional[JSONLike]:
         :return: the tx, if present
         """
         # Cosmos does not distinguish between transaction receipt and transaction
-        tx_receipt = self._try_get_transaction_receipt(tx_digest)
-        return tx_receipt
+        tx_with_receipt = self._try_get_transaction_with_receipt(tx_digest)
+        return tx_with_receipt
```

Comment:
Is this correct? Isn't this include the TX and receipt together.

Shouldn't it be `tx_with_receipt.get("txResponse")` similar to `get_transaction_receipt` above?